beaker.gemspec

```
@@ -39,6 +39,7 @@ Gem::Specification.new do |s|
 
   s.add_runtime_dependency 'hocon', '~> 1.0'
   s.add_runtime_dependency 'net-ssh', '>= 5.0'
+  s.add_runtime_dependency 'ed25519' # net-ssh needs this to connect to modern systems
```

Do we want to limit this? And I wonder about this phrasing, even though it's a tiny difference.
```suggestion
  s.add_runtime_dependency 'ed25519', '~> 1.0' # net-ssh compatibility with ed25519 keys
```